modules/samples/artifacts/0049/keyvalue-mapper-sample.siddhi

```diff
@@ -0,0 +1,21 @@
+@App:name("keyvalue-mapper-sample")
+
+@source(type='tcp', context='SmartHomeData', port='9892',
+@map(type='binary'))
+define stream SmartHomeData (houseId int, maxVal float, minVal float, avgVal double);
+
+@sink(type='inMemory', topic='home', @map(type='keyvalue', 
+@payload(a="""houseId""",b="""maxVal""", c ="""minVal""", d="""avgVal""")))
+define stream InMemorySmartHomeInputData (houseId int, maxVal float, minVal float, avgVal double);
+
+@source(type='inMemory', topic='home', @map(type='keyvalue', @attributes(houseId ="a", maxVal = "b", minVal = "c", avgVal = "d")))
```

Comment:
Why do you need this in-memory source?

Reply:
I add this to show the use of Source Mapper

Reply:
please fix as discussed